cli/src/image_data.js

```diff
@@ -0,0 +1,12 @@
+export default class ImageData {
+  constructor(data, width, height) {
+    // Need to manually copy the memory as wasm-bindgen does not by default
+    // and by the time we get control in JS land, the memory has already
+    // been corrupted.
+    // FIXME: This is bad because it’s overhead that we should only need
```

Comment:
This is only necessary for `rotate.rs` because we do manual management there; any other Rust codecs will be fine, just like C++. Seems easy enough to move this copying only to `rotate`'s JS wrapper.

Reply:
Not sure how without it being overwritten by every rebuild of rotate. But I’ll open an issue to fix this.

Reply:
You have JS binding above, where you grow memory and so on - can't you just put `new Uint8ClampedArray(...)` in there?

Reply:
JS is not autogenerated for rotate, we have custom one.

Reply:
Resize JS code already does `new Uint8ClampedArray`, so can just remove here.

Reply:
I think comments can be removed now as well.